Lamouresparus/Asa
# Asa 35 display and edit profile data

### Tracked by [JIRA-35](https://loveotudor.atlassian.net/browse/ASA-35)
 
### This PR adds

- display user profile data
- edit user profile data

### Considerations and implementation

we converted user object to string with gson so we can save to shared prefs, to avoid  too many calls to saving all of the user attributes  to pref


### How to Test

1. Launch the App
2. On the home screen, click on the "Profile Icon": should see a new screen displaying the user's profile with it's correct data. You can also edit the user data

### Test Added
N/A
